lib/src/room.dart

```diff
@@ -291,6 +291,11 @@ class Room {
       if (sender != null) return sender;
     }
     if (membership == Membership.leave) {
+      if (directChatMatrixID != null) {
```

Comment:
I would merge this with the lines just below this, looks like prev_sender is never sent in member events